app/src/lang/translations/en-US.yaml

```
@@ -2476,10 +2476,16 @@ operations:
     name: Send Email
     description: Send an email to one or more people
     to: To
-    to_placeholder: Add e-mail addresses and press enter...
+    to_placeholder: Add email addresses and press enter...
     body: Body
     template: Template
     data: Data
+    cc: CC
+    cc_placeholder: Add CC email addresses and press enter...
+    bcc: BCC
+    bcc_placeholder: Add BCC email addresses and press enter...
+    replyTo: Reply To
+    replyTo_placeholder: Add reply-to email address...
```

Thank you for the PR!

Translation keys should be snake case for consistency.
I think it could be either `reply_to` or `replyto`.
This change will have to be updated in other files too.

```suggestion
    reply_to: Reply To
    reply_to_placeholder: Add reply-to email address...
```

---

Thx, I also updated the keys in the operator.